src/openrct2/game.h

```diff
@@ -150,6 +150,7 @@ extern bool gInUpdateCode;
 extern bool gInMapInitCode;
 extern sint32 gGameCommandNestLevel;
 extern bool gGameCommandIsNetworked;
+extern char gCurrentLoadedPath[260];
```

Comment:
Because I don't allow `windows.h` to be included in any header file, it causes too many issues across source files and over-pollutes the namespace with macros and other things.